nestkernel/mpi_manager.cpp

```diff
@@ -716,14 +716,12 @@ nest::MPIManager::grng_synchrony( unsigned long process_rnd_number )
 // any_true: takes a single bool, exchanges with all other processes,
 // and returns "true" if one or more processes provide "true"
 bool
-nest::MPIManager::any_true( bool my_bool )
+nest::MPIManager::any_true( const bool my_bool )
 {
   if ( get_num_processes() == 1 )
     return my_bool;
 
-  int my_int = 0;
-  if ( my_bool )
-    my_int = 1;
+  int my_int = my_bool;
```

Comment:
I think it would be useful to add a comment here, explaining why you need to convert to an int here. Two years from now a reader may not remember that MPI does not support bool. The declaration can also be `const`.

Then on line 734 below (I cannot add comments there in Github), I think it would be even safer to check `all_int[i] != 0`)  instead of checking for `== 1`.